updated version of OpenGEE to 5.3.6

tst-nfarah opened the pull request:
This fixes #1814

tst-nfarah committed:
updated version of opengee to 5.3.6

Commit:
update urls to use correct version

tst-mbiancalana reviewed (approved):
These changes look good!

tst-nfarah commented:
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogle%2Fearthenterprise%2Fpull%2F1815) for more info**.

@googlebot I fixed it
